server/controllers/googleApiController.js

```diff
@@ -0,0 +1,27 @@
+const { google } = require("googleapis");
+
+const oAuth2Client = new google.auth.OAuth2(
+  "871373961261-rjej65g97dc3o6jiuflq6s2gp5v9ptut.apps.googleusercontent.com",
```

Comment:
can you store these in env variables?

Reply:
Sure, i will move all credentials in the server side to env once I have the GET events working as well